README.md

```diff
@@ -33,15 +33,81 @@ val metadata: IO[RecordMetadata] = producer.use { producer =>
 ## Consumer usage example
 
 ```scala
+import cats.data.{NonEmptySet => Nes}
+
 val consumer = Consumer.of[IO, String, String](config, ecBlocking)
 val records: IO[ConsumerRecords[String, String]] = consumer.use { consumer => 
   for {
-    _       <- consumer.subscribe(Nel("topic"), None)
+    _       <- consumer.subscribe(Nes("topic"))
     records <- consumer.poll(100.millis)
   } yield records 
 }
 ```
 
+## Handling consumer group rebalance
+It's possible to provide a callback for a consumer group rebalance event, which can be useful if you want to do some computations,
+save the state, commit some offsets (or do anything with the consumer before the new group is applied).  
```

Comment:
`before the new group is applied`
let's change it to
`before the new group assignment is applied` ?
(I'm not a native english speaker, so mb suggesting nonsense)

Reply:
> before the new group is applied

I don't thinks this is a right wording in general, I'd say something like "before partitions rebalance is finalised"

Reply:
btw when I was writing scaladocs for it I was using [ConsumerRebalanceListener](https://kafka.apache.org/25/javadoc/index.html?org/apache/kafka/clients/consumer/ConsumerRebalanceListener.html) for inspiration.
I think we want to be as close as possible to terminology used in kafka java consumer, right?

Reply:
Maybe `whenever partition assignment changes`? That's a phrase from the official documentation for `ConsumerRebalanceListener`.